handlers/assign_reviewer/__init__.py

```diff
@@ -55,7 +55,7 @@ def on_pr_opened(self, api, payload):
         pr = payload["pull_request"]
         # If the pull request already has an assignee,
         # don't try to set one ourselves.
-        if pr["assignee"] != None:
+        if pr["assignee"] != None:      # NOQA (silence flake8 here)
```

Comment:
flake8 prefers `is not None`, but we need `!=` to call our overridden `__ne__` method (the former checks for an exact match of the object, which will fail while testing)